addon/lib/field.js

```diff
@@ -72,21 +73,50 @@ export default Base.extend(Evented, {
   saveDocumentDateAnswerMutation,
   saveDocumentTableAnswerMutation,
 
+  /**
+   * The Apollo GraphQL service for making requests
+   *
+   * @property {ApolloService} apollo
+   * @accessor
+   */
```

Comment:
In the lib we only want the public API or complex private methods/properties documented, this is kind of obvious what it is.